frontend/src/components/search-bar/SearchBar.jsx

```diff
@@ -19,7 +19,7 @@ function SearchBar(props) {
                     type="text"
                     onChange={e => props.setQuery(e.target.value)}
                     value={props.query}
-                    placeholder="Search for a camera"
```

Comment:
I had it explicitly as "cameras" since at the moment there's no way to add error handling to this initial search. A user can use our product and search "toys" and we'll return a legit list of products under the "toys" category.

So I thought as "camera" explicitly being mentioned, users will be more inclined to search for a camera rather than anything that exists on Google Shopping

Reply:
I think because we're trying to generalize our project for the pitch, we should approach all placeholder text in that mindset too. Therefore I like the removal of "cameras" because it generalizes it to be all products